src/services/event.ts

```diff
@@ -164,7 +164,12 @@ async function findEventHelperBlob({
       'regionId',
       'data',
     ],
-    raw: true,
+    include: [
+      {
```

Comment:
we should check if we need to sort these a particular way

Reply:
There is an order by that Jon added based on the orig ticket.

![image](https://github.com/HHS/Head-Start-TTADP/assets/84350609/15ed2514-1c01-4060-873f-ea15df9abbf2)

Reply:
Added sort to the sessions as well.